src/app.module.ts

```diff
@@ -1,9 +1,12 @@
 import { Module } from '@nestjs/common';
 import { ConfigModule, ConfigService } from '@nestjs/config';
+import { JwtModule } from '@nestjs/jwt';
+import { PassportModule } from '@nestjs/passport';
 import { TypeOrmModule } from '@nestjs/typeorm';
 
```

Comment:
Remove this line.

Reply:
Resolved on 298ab1e4e2ae083bd4d5450e9bc709597987671c.